From 9dd0685cb509235560aee4ece73389e47c6f54cd Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Sun, 28 Jun 2026 21:40:17 +0800
Subject: [PATCH 429/441] AOSCOS: net/phytium: drop of_gpio.h inclusion

This seems used nowhere and the header is removed in Linux 7.1.

Signed-off-by: Xi Ruoyao <xry111@xry111.site>
---
 drivers/net/ethernet/phytium/phytmac_main.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/ethernet/phytium/phytmac_main.c b/drivers/net/ethernet/phytium/phytmac_main.c
index 3f1f2098284d..0d6bdb44cf51 100644
--- a/drivers/net/ethernet/phytium/phytmac_main.c
+++ b/drivers/net/ethernet/phytium/phytmac_main.c
@@ -28,7 +28,6 @@
 #include <linux/phy.h>
 #include <linux/of.h>
 #include <linux/of_device.h>
-#include <linux/of_gpio.h>
 #include <linux/of_mdio.h>
 #include <linux/of_net.h>
 #include <linux/ip.h>
-- 
2.55.0

